README.md

```diff
@@ -1,29 +1,71 @@
 # ember-service-worker-update-notify
 
-Adds an `{{service-worker-update-notify}}` component which displays
+Adds a `<ServiceWorkerUpdateNotify />` component which displays
 a reload link if the service-worker has found an update.
 
 Overwrite the default message using the component in block form:
 
 ```handlebars
 <ServiceWorkerUpdateNotify>
-  <a class="service-worker-update-notify" href="">
+  <a class="service-worker-update-notify" href={{this.router.currentURL}}>
     A new version is available, click here to update.
   </a>
 </ServiceWorkerUpdateNotify>
 ```
 
-The poll interval can be configured with:
+Additionally, the frequency of how often a check will be made for an update
+can be configured via the `pollingInterval` argument.
+The time is measured in milliseconds. The default is 1200000 (20 minutes).
+
 ```handlebars
-<ServiceWorkerUpdateNotify @pollingInterval={{yourTimeout}}/>
+<ServiceWorkerUpdateNotify @pollingInterval={{2000}}>
+  <a class="service-worker-update-notify" href={{this.router.currentURL}}>
+    A new version is available, click here to update.
+  </a>
+</ServiceWorkerUpdateNotify>
+```
+
+## Testing in Your App
+
+Testing this in your app should mainly be concerned with presence
+and and what it looks like.
 
+During testing, the polling will be disabled,
+and the reveal of the "New version available" content
+is controlled by a promise set on the `window`.
+In your tests, two helpers will aid you in asserting
+presence and appearance: `setupServiceWorkerUpdater`,
+and `hasServiceWorkerUpdate`.
+
+```js
+// ...
+import {
+  setupServiceWorkerUpdater,
+  hasServiceWorkerUpdate
+} from 'ember-service-worker-update-notify/test-support/updater';
+
+module('Application | Index', function(hooks) {
+  setupApplicationTest(hooks);
+  setupServiceWorkerUpdater(hooks);
+
+  test('the update is shown', async function(assert) {
+    // assert that the content is not shown
+
+    await serviceWorkerUpdate();
+
+    // assert that the content is shown
+
+  });
+});
 ```
 
+
 ## Installation
 
 ```bash
-ember install ember-concurrency
-ember install ember-service-worker-update-notify
+yarn add --dev ember-concurrency
+yarn add --dev ember-concurrency-test-waiter
+yarn add --dev ember-service-worker-update-notify
```

Comment:
Why that change? `ember install` will use `yarn` if used by the app, and run the addon's default blueprints, which might be required. Not for these addons, but in general it seems to be the safer choice to use `ember install` I think!?

Reply:
My thinking behind this was to de-mystify what ember install actually does. For 90+% of addons, it's not needed, and when developing an addon, it's actually wrong, because sometimes you need things in `dependencies`.